store/sql_session_store.go

```diff
@@ -255,3 +255,32 @@ func (me SqlSessionStore) UpdateDeviceId(id, deviceId string) StoreChannel {
 
 	return storeChannel
 }
+
+func (me SqlSessionStore) AnalyticsSessionCount(teamId string) StoreChannel {
+	storeChannel := make(StoreChannel)
+
+	go func() {
+		result := StoreResult{}
+
+		query :=
+			`SELECT
+                COUNT(*)
+            FROM
+                Sessions`
+
+		if len(teamId) > 0 {
+			query += " WHERE TeamId = :TeamId"
```

Comment:
This will include expired sessions?  if a user never logs back in it won't get deleted from the db.